### PR TITLE
Fix warnings on Rails boot

### PR DESCRIPTION
**Pull request by Gargron**

### CODE REVIEW

File: config/initializers/sidekiq.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative '../../app/lib/sidekiq_error_handler'
```
Comment (ClearlyClaire) [inline config/initializers/sidekiq.rb:3]: Uh, why is that necessary? Shouldn't it get auto-loaded when mentioning `SidekiqErrorHandler`?

Comment (Gargron) [reply config/initializers/sidekiq.rb:3]: Because Rails complains about autoloading during initialization

Comment (ClearlyClaire) [reply config/initializers/sidekiq.rb:3]: ok. I would move outside `app` then, to `lib/`

**Review by ykzts — APPROVED** [review]

File: config/initializers/sidekiq.rb (hunk 1 above)
Comment (ClearlyClaire) [reply config/initializers/sidekiq.rb:3]: looks good to me if this is moved to `lib/`

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]

**Review by ClearlyClaire — APPROVED** [review]
